### ISSUE TITLE
Filter issue: Critical biodiversity area and ecosystem should have a select all function and that should be the default

### ISSUE THREAD

**Issue by LunaAsefaw** [kartoza/sawps#1396]
When users land on the explore page, none of the critical biodiversity areas or ecosystems are selected by default. Additionally, the ecosystems filter has many options so selecting one by one might not be efficient and I am worried that properties will not be shown on the map if none of the areas/ecosystems are selected by default.
![image](https://github.com/kartoza/sawps/assets/90245715/28dd3583-09ae-4cea-86f3-a4a428b8bbed)

For consistency (since this is the approach we have with all of the filters), it should have a select all option for both these filters and it should be selected by default so all the properties are included at first (since the properties displayed on the map are based on the spatial filters too) then users can deselect as they wish.

**Comment by danangmassandy** [comment]
@LunaAsefaw I think the behavior of spatial filter would be similar to activity filter. When all items are selected by default, then we don't need to apply the filter. What do you think?

**Comment by LunaAsefaw** [comment]
But if all the items are selected by default but there are no ticks showing that, I think it would be misleading for the user because they wouldn't know that they are all selected by default when all the layers are ticked off instead of checked. For example, I can tell that all the layers are being used right now even those that are not checked but the users might think those are properties that do not fall on the areas selected since they are being returned when the ecosystem type is not selected. What do you think?

**Comment by dimasciput** [comment]
@LunaAsefaw @amyburness 

While other filters might make sense, I don't believe the subtractive filter is the best choice here given the numerous options available. It might be more effective to leave them unchecked and simply display all properties. This is because there's a chance some properties don't have spatial data, a lesson we've learned from FBIS.

From a UX perspective, if a user wants to view only Albany, they would first need to uncheck all spatial data under critical biodiversity and ecosystem type, and then select Albany.

Btw, I also don't completely agree with the default selected values for the other filters. I believe it's more effective to leave them unchecked and display all data as if they were checked. Look at GBIF, for instance, its filtering is more user-friendly. However, since we've already implemented this approach, let's not change it.
